hack/errata.py

```diff
@@ -83,13 +90,70 @@ def notify(message, webhook=None):
         print(message)
         return
 
+    msg_text = '<!subteam^STE7S7ZU2>: {fulladvisory} shipped {when}: {synopsis}'.format(**message)
+    if  message['approved_pr']:
+        msg_text += "\nPR {approved_pr} has been approved".format(**message)
+
     urllib.request.urlopen(webhook, data=urllib.parse.urlencode({
         'payload': {
-            'text': '<!subteam^STE7S7ZU2>: {fulladvisory} shipped {when}: {synopsis}'.format(**message),
+            'text': msg_text,
         },
     }).encode('utf-8'))
 
 
+def get_open_prs_to_fast(repo):
+    for pr in repo.get_pulls(state='open', sort='created', base='master'):
+        try:
+            # Skip unknown PRs
+            if not pr.title.startswith("Enable "):
+                continue
+            # Ignore PRs which don't target fast
+            if pr.title.split(" ")[3] != "fast":
+                continue
+            yield pr
+        except Exception as e:
+            print(f"Failed to parse {pr.number}: {e}")
+
+
+def extract_errata_number_from_body(body):
+    ERRATA_MARKER = 'https://errata.devel.redhat.com/advisory/'
+    first_line = body.split('\n')[0]
+    links = [
+        x for x in first_line.split(' ') if x.startswith(ERRATA_MARKER)
+    ]
+    if len(links) == 0:
+        print("No links found in PR body: {}".format(body))
+        return None
+    errata_num = links[0].rsplit('/', 1)[-1]
+
+    try:
+        return int(errata_num)
+    except ValueError:
+        print("Failed to convert PR number to int: {}".format(errata_num))
+        return None
+
+
+def lgtm_fast_pr_for_errata(githubrepo, message):
+    github_token = os.environ.get("GITHUB_TOKEN")
```

Comment:
Can we pass this in from an argparse arg too, following the `webhook` pattern?

Reply:
Fixed in 0c648c9d184320b39b668223eb69fdf7e890ded4